pkgs/by-name/bi/birdtray/package.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, libsForQt5
+, fetchpatch
+, thunderbird
+}:
+
+stdenv.mkDerivation rec {
+  pname = "birdtray";
+  version = "1.11.4";
+
+  src = fetchFromGitHub {
+    owner = "gyunaev";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-rj8tPzZzgW0hXmq8c1LiunIX1tO/tGAaqDGJgCQda5M=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    libsForQt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libsForQt5.qtbase
+    libsForQt5.qttools
+    libsForQt5.qtx11extras
+  ];
+
+  cmakeFlags = [
+    (lib.cmakeFeature "OPT_THUNDERBIRD_CMDLINE" "thunderbird") # get thunderbird from PATH
+  ];
+
+  patches = [
+    (fetchpatch {
+      name = "fix-path-handling.patch";
+      url = "https://github.com/gyunaev/birdtray/commit/54b304d92188429792c264b07ff45897699f2d3e.patch";
+      hash = "sha256-ME635Kt1b9RJKCqtAZBFa93OIA0u2Z4tWIlGcI374j0=";
+    })
+  ];
+
+  qtWrapperArgs = [
+    "--prefix PATH : ${lib.makeBinPath [ thunderbird ]}"
```

Comment:
What is more important than the formatting is, if we want to hardcode thunderbird here as that doesn't play that nicely with programs.thunderbird and it would probably be better to take it from the users PATH.

Reply:
Fair enough, I went back and forth on this, I can see arguments for both. Anyway, updated.